tools/cassandra/handler.go

```diff
@@ -191,12 +194,41 @@ func newCQLClientConfig(cli *cli.Context) (*CQLClientConfig, error) {
 		}
 	}
 
+	config.AddressTranslator = &c.CassandraAddressTranslator{
+		Translator: cli.GlobalString(schema.CLIOptAddressTranslator),
+		Options:    parseOptionsMap(cli.GlobalString(schema.CLIOptAddressTranslatorOptions)),
+	}
+
 	if err := validateCQLClientConfig(config); err != nil {
 		return nil, err
 	}
 	return config, nil
 }
 
+func parseOptionsMap(value string) map[string]string {
+	if len(value) == 0 {
+		return make(map[string]string)
+	}
+
+	split := strings.Split(value, ",")
+
+	parsedMap := make(map[string]string)
+
+	for _, pair := range split {
+		trimmedPair := strings.ReplaceAll(pair, " ", "")
+		if len(trimmedPair) == 0 {
+			continue
+		}
+		splitPair := strings.Split(trimmedPair, "=")
+		if len(splitPair) != 2 {
+			continue
+		}
+		parsedMap[splitPair[0]] = splitPair[1]
```

Comment:
`key1=,=value2` will parse to 2 paris key1='', and ''=value2.

Reply:
fixed, I just ignored pairs for which key or value is empty string